src/modules/pages/components/RouteLayouts/Default/Default.css

```diff
@@ -9,7 +9,7 @@
 
 .coloniesList {
   flex: 0 0 auto;
-  width: 50px;
+  width: 60px;
```

Comment:
I feel like this is a bad change. 

Note that increasing this has a cascading effect over the whole layout of the app as a lot of pages do width `calc` assuming this 50px. So if you change it here make sure you redo all those other places that this influences.

Right now, it broke the full page loading layout:

![Screenshot from 2021-02-05 15-12-56](https://user-images.githubusercontent.com/1193222/107038233-e2b9a400-67c4-11eb-8823-67642c9aa69c.png)

Notice the scroll bar at the bottom!

Reply:
I will see if adjusting all the other places isn't too much problem then

Reply:
Tested it at 50px, it's not this change that's breaking the page

Reply:
Hell, lemme tell you something. This is even happening in `master` lol

Tried to look at master to see where's the difference, but there's no difference

Reply:
I'll test it again

Reply:
Yeah, you can leave it as it is for now, I'll test it if something actually breaks when I rebase my #2359 then I'll fix it there